core/src/main/java/com/sequenceiq/cloudbreak/converter/SmartSenseSubscriptionToSmartSenseSubscriptionJsonConverter.java

```diff
@@ -10,7 +14,8 @@
         extends AbstractConversionServiceAwareConverter<SmartSenseSubscription, SmartSenseSubscriptionJson> {
 
     @Override
-    public SmartSenseSubscriptionJson convert(SmartSenseSubscription source) {
```

Comment:
this is does not makes sense. in case of the the source object is null then the spring conversion logic can not detect the correct converter..